looper/conductor.py

```diff
@@ -767,6 +772,31 @@ def _reset_curr_skips(self):
         self._curr_skip_pool = []
         self._curr_skip_size = 0
 
+    def check_executable_path(self, pl_iface):
+        """Determines if supplied pipelines are callable.
```

Comment:
run linter, because this comment is incorrect. should be 
```
"""
This is comment...
"""
```
+ add typing to the pl_iface and :param to docstring